cms/djangoapps/contentstore/helpers.py

```diff
@@ -454,78 +471,122 @@ def _import_files_into_course(
     course_key: CourseKey,
     staged_content_id: int,
     static_files: list[content_staging_api.StagedContentFileData],
-) -> StaticFileNotices:
+    usage_key: UsageKey,
+) -> (StaticFileNotices, dict[str, str]):
```

Comment:
```suggestion
) -> tuple[StaticFileNotices, dict[str, str]]:
```

Reply:
BTW, I am absolutely going to accept these suggestions just as soon as CI finishes running.